public.tf

```diff
@@ -1,11 +1,11 @@
 module "public_label" {
   source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.2"
   namespace  = "${var.namespace}"
-  name       = "${var.availability_zone}"
-  attributes = ["public"]
+  name       = "${var.name}"
   stage      = "${var.stage}"
   delimiter  = "${var.delimiter}"
   tags       = "${var.tags}"
+  attributes = ["${compact(concat(var.attributes, list("public")))}"]
```

Comment:
I think, it should be `["${compact(concat(var.attributes, list("${var.availability_zone}", "public")))}"]`.
In this way, we will be sure, that subnet label contains AZ name.

Reply:
`var.availability_zone` could be provided in the `attributes` by the user of the module if needed, as well as any other attributes the client wants or needs.
What if I don't want the AZ name to be present in the subnet name?
We probably should not make the `subnets` module to force a specific naming convention on the clients.